src/util/Util.ts

```diff
@@ -12,4 +12,33 @@ import { Readable } from 'stream';
  */
 export const ensureTrailingSlash = (path: string): string => path.replace(/\/*$/u, '/');
 
+/**
+ * Joins all strings of a stream.
+ * @param stream - Stream of strings.
+ *
+ * @returns The joined string.
+ */
 export const readableToString = async(stream: Readable): Promise<string> => (await arrayifyStream(stream)).join('');
```

Comment:
Might want to change the very generic `Util.ts` into specific `StreamUtil`, `ContentTypeUtil` etc.

Reply:
Will wait until the file store is also merged to do that to reduce the amount of merge conflicts that would generate